vector/src/main/java/im/vector/riotx/features/settings/VectorSettingsGeneralFragment.kt

```diff
@@ -771,44 +772,23 @@ class VectorSettingsGeneralFragment : VectorSettingsBaseFragment() {
                     val oldPwd = oldPasswordText.text.toString().trim()
                     val newPwd = newPasswordText.text.toString().trim()
 
-                    notImplemented()
-                    /* TODO
                     showPasswordLoadingView(true)
-
-                    session.updatePassword(oldPwd, newPwd, object : MatrixCallback<Unit> {
-                        private fun onDone(@StringRes textResId: Int) {
+                    session.changePassword(oldPwd, newPwd, object : MatrixCallback<Unit> {
```

Comment:
A homeserver may not support users changing their password (e.g. if is configured to perform authentication against an external system a.k.a SSO)
-> check for m.change_password in /_matrix/client/r0/capabilities